docs/samples/kml/World_Map.kml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<kml xmlns:kml="http://earth.google.com/kml/2.2">
+<kml xmlns="http://www.opengis.net/kml/2.2">
   <Document id="featureCollection">
```

Comment:
this is outdated for the namespace. 
fastkml will read only registered namespaces, https://fastkml.readthedocs.io/en/main/configuration.html#registering-additional-namespaces